README.md

```diff
@@ -1 +1,57 @@
-# code-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MObile First Portfolio
```

Comment:
Please change the title on the Readme. It totally shows you've copied from another project

Reply:
> Please change the title on the Readme. It totally shows you've copied from another project

- [ ] Please I think it would be better here to say: '' It totally shows you've copied from another project'', but rather ask him in a polite and friendly way to update his readme file. And tell why it is a good thing to have a well-documented readme file.